rust/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "hacl"
-version = "0.0.1-dev.1"
+version = "0.0.2-dev.1"
```

Comment:
question: Should the hacl/hacl-sys versions be aligned? `0.0.2-dev.1` vs `0.0.2-dev.3`.

Reply:
That's not really necessary for these dev versions.